list.js

```diff
@@ -0,0 +1,61 @@
+class Component {
```

Comment:
move the file to src/components/currentRecipe.js
```suggestion
class CurrentRecipe {
```

Reply:
this is still to be fixed